sleepyq/__init__.py

```diff
@@ -36,66 +36,208 @@ def __init__(self, data):
         super(Sleeper, self).__init__(data)
         self.bed = None
 
+class Light(APIobject):
+    def __init__(self, data):
+        super(Light, self).__init__(data)
+        self.bed = None
+
+class FavSleepNumber(APIobject):
+    def __init__(self, data):
+        super(FavSleepNumber, self).__init__(data)
+        self.bed = None
+        self.left = None
+        self.right = None
+
 class Sleepyq:
     def __init__(self, login, password):
         self._login = login
         self._password = password
         self._session = requests.Session()
+        self._api = "https://api.sleepiq.sleepnumber.com/rest"
+
+    def __makeRequest(self, url, mode="get", data=""):
```

Comment:
Doesn't python tend to favor snakecase, ie `__make_request`?

Reply:
changed to snakecase, should I change set_favsleepnumber and get_favsleepnumber as well?